portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -37,29 +36,25 @@
 
   private List<String> messages = new ArrayList<>();
   private static final String SPACE = " ";
+  private String numberChoiceString = "";
```

Comment:
Is this being used anywhere? If not, we could delete it.

Reply:
I think this was also for some earlier debugging. I will delete it!